testing/sdk_cmd.py

```diff
@@ -22,7 +22,7 @@
 log = logging.getLogger(__name__)
 
 DEFAULT_TIMEOUT_SECONDS = 30 * 60
-
+LINUX_USER = os.getenv("DCOS_SSH_USERNAME", "core")
```

Comment:
Same comment as for `sdk_auth` -- specifically about **only** specifying it here.